src/server.rs

```diff
@@ -611,7 +612,10 @@ impl LanguageServer for LspServer {
                 if let flux::semantic::nodes::Expression::Member(member) = callee.clone() {
                     let name = member.property.clone();
                     if let flux::semantic::nodes::Expression::Identifier(ident) = member.object.clone() {
-                        signatures.extend(find_stdlib_signatures(name, ident.name.to_string()));
+                        // XXX: rockstar (11 Jan 2022) - Symbol should be able to
+                        // be .into() String. When that happens, this `format!`
+                        // can go away.
+                        signatures.extend(find_stdlib_signatures(format!("{}", name), ident.name.to_string()));
```

Comment:
Dang, I was hoping you already had a strategy in mind for addressing this. I'll fix this today.

Reply:
I have come around to using `.into()` where I would normally `.to_string()`, but that's mostly a personal preference. I think a followup PR with the change to the `find_stdlib_signatures` is the right thing.

Reply:
Yah, I use `into()` quite a bit as well for `String`, though it may fail inference sometimes which can be annoying